Hi @larshg 
Sorry for the long delay.  
Could you please look into the following problems?

The features build failed on x64-windows triplet:

The feature `all` is broken due to this:
```
CMake Error at CMake/vtkDownload.cmake:8 (message):
  Attempted to download Module_MomentInvariants when VTK_FORBID_DOWNLOADS is
  ON
Call Stack (most recent call first):
  CMake/vtkModuleRemote.cmake:142 (vtk_download_attempt_check)
  Remote/MomentInvariants.remote.cmake:5 (vtk_fetch_module)
  Remote/CMakeLists.txt:9 (include)
```
The feature `qt, atlmfc, openvr ,paraview` is broken due to this:
```
CMake Error at ThirdParty/hdf5/CMakeLists.txt:26 (message):
  An external MPI-aware HDF5 requires that VTK be built with MPI support as
  well.
```
The feature `python` is broken due to this:
```
CMake Error at CMake/vtkModule.cmake:4140 (message):
  Could not find the Python3 external dependency.
Call Stack (most recent call first):
  Utilities/Python/CMakeLists.txt:49 (vtk_module_find_package)
```

The feature `mpi` is broken due to this:
```
CMake Error at CMake/vtkModule.cmake:959 (message):
  The VTK::IOParallelNetCDF module requires the disabled module VTK::mpi.
Call Stack (most recent call first):
  CMakeLists.txt:251 (vtk_module_scan)
```

--- a/ports/vtk/CONTROL
+++ b/ports/vtk/CONTROL
@@ -36,6 +36,10 @@ Feature: cuda
 Description: Support CUDA compilation
 Build-Depends: cuda
 
+Feature: opengl
+Description: All opengl related modules
+Build-Depends: vtk[core]
+
 Feature: all
-Description: Build vtk modules required by paraview
+Description: Build all vtk modules
 Build-Depends: libmysql, ffmpeg, gdal, vtk[core, qt, python, mpi], vtk[core, atlmfc] (windows)

--- a/ports/vtk/portfile.cmake
+++ b/ports/vtk/portfile.cmake
@@ -270,8 +270,7 @@ file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/share)
 
 # =============================================================================
 # Handle copyright
-file(COPY ${SOURCE_PATH}/Copyright.txt DESTINATION ${CURRENT_PACKAGES_DIR}/share/vtk)
-file(RENAME ${CURRENT_PACKAGES_DIR}/share/vtk/Copyright.txt ${CURRENT_PACKAGES_DIR}/share/vtk/copyright)
+file(INSTALL ${SOURCE_PATH}/Copyright.txt DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
 
 vcpkg_copy_tool_dependencies("${CURRENT_PACKAGES_DIR}/tools/vtk")
 

--- a/ports/vtk/portfile.cmake
+++ b/ports/vtk/portfile.cmake
@@ -82,6 +82,19 @@ if("mpi" IN_LIST FEATURES)
     )
 endif()
 
+if("opengl" IN_LIST FEATURES)
+    list(APPEND ADDITIONAL_OPTIONS
+        -DVTK_MODULE_ENABLE_VTK_DomainsChemestryOpenGL2=YES
+        -DVTK_MODULE_ENABLE_VTK_ImagingOpenGL2=YES
+        -DVTK_MODULE_ENABLE_VTK_RenderingContextOpenGL2=YES
+        -DVTK_MODULE_ENABLE_VTK_RenderingGL2PSOpenGL2=YES
+        -DVTK_MODULE_ENABLE_VTK_RenderingLICOpenGL2=YES
+        -DVTK_MODULE_ENABLE_VTK_RenderingOpenGL2=YES
+        -DVTK_MODULE_ENABLE_VTK_RenderingVolumeOpenGL2=YES
+        -DVTK_MODULE_ENABLE_VTK_opengl=YES
+        )
+endif()
+
 if("all" IN_LIST FEATURES)
     list(APPEND ADDITIONAL_OPTIONS
         -DVTK_USE_TK=OFF # TCL/TK currently not included in vcpkg

